Monika After Story/game/script-ch30.rpy

```diff
@@ -536,21 +580,12 @@ label ch30_autoload:
     #Run actions for any events that are based on the clock
     $ evhand.event_database=Event.checkCalendar(evhand.event_database)
 
-    #Skip all greetings if you closed the game on Monika
-    if persistent.closed_self:
-        #pick a random greeting
-        if is_monika_in_room:
```

Comment:
FYI, greetings should be the last thing pushed onto the event stack prior to the jump to `ch30_loop`. Just set a local var to the selected greeting in the checks above and have it be pushed onto Event stack here if the greeting is not None